src/main/java/org/apache/datasketches/sampling/PreambleUtil.java

```diff
@@ -102,7 +102,7 @@
  *  </pre>
  *
  * <p><strong>VarOpt Union:</strong> VarOpt unions also store more information than a reservoir
- * sketch. As before, we keep values with similar o hte same meaning in corresponding locations
+ * sketch. As before, we keep values with similar to the same meaning in corresponding locations
```

Comment:
You have multiple sketches using the same PreambleUtil class.  That is OK, but where you have the documentation for each of the specific sketches that use this class is confusing.  You chose to place the preamble structure for Ebpps in its Sketch class, but for VarOpt and Reservoir is in this class.  You could either move the specific preamble structures in to their respective sketch classes, or (I think I would prefer) have all of the structure docs in this class, separated by clear documentation boundaries.  I.e., use of <h2> </h2> headers, which work great in javadocs.

Reply:
I was unsure of the best way to do this. I feel like stacking them all together is not actually helpful since the class starts to feel bloated. You seem less worried about that?

Reply:
If the sketch specific docs, and related code is nicely separated out with headers, it won't feel bloated.  It all has a reason for being there.   A class begins to feel "bloated", I think, when too much totally unrelated stuff is put into the same class.  

The intention was for the PreambleUtil class to be THE place to go to understand the various serialization formats and hold the static constants and static helper methods related to serialization. Having this stuff spread out in different classes makes it hard to find -- 'cause I, at least, will not be able to remember where to go to find it.  We have this inconsistency in some other sketches as well.

Reply:
The challenge here is that I didn't want to create a stand-alone family, so there are now 3 separate sketches squashed into the same `PreambleUtil` class. But that's fine, I certainly don't have an objection to moving it.